src/index.html

```diff
@@ -2,20 +2,74 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta 
+    <meta
       name="viewport"
       content="
-        width=device-width, 
-        user-scalable=no, 
-        initial-scale=1.0, 
-        maximum-scale=1.0, 
+        width=device-width,
+        user-scalable=no,
+        initial-scale=1.0,
+        maximum-scale=1.0,
         minimum-scale=1.0
-      "
-    >
+    ">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <div class="product card__product">
+        <img
+          src="images/imac.jpeg"
+          alt="product photo"
+          class="product__photo"
+        >
+      </div>
+      <div class="product-info">
+        <h2 class="product-info__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="product-info__code">
+          Product code. 195434
+        </p>
+
+        <div class="product-rating">
+          <ul class="stars stars--4 product-rating__stars">
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+          </ul>
+
+          <p class="product-rating__review">
+            Reviews: 5
+          </p>
+        </div>
+        <div class="product-info__price">
+          <p class="product-info__price-text">
+            Price:
+          </p>
+          <p class="product-info__price-tag">
+            $2,199
+          </p>
+        </div>
+      </div>
+
+      <div class="card-button">
```

Comment:
```suggestion
      <div class="card__button">
```
It can be just an element of block card